src/components/panel/panel.js

```diff
@@ -903,7 +901,7 @@ MdPanelRef.prototype.attachOnly = function() {
 
 
 /**
- * Detaches the panel. Will hide the panel first if visible.
+ * Only detaches the panel. Will NOT hide the panel first.
```

Comment:
Consider also changing the name to detachOnly. Otherwise we should remove the Only from attach.